.github/workflows/pre_commit_suggestions.yaml

```diff
@@ -60,3 +60,16 @@ jobs:
           cat ./diff | \
           GITHUB_EVENT_PATH=./event \
             reviewdog -f=diff -f.diff.strip=1 -reporter=github-pr-review
+
+  delete-stale-suggestions:
+    if: |
+      github.event.workflow_run.event == 'pull_request' &&
+      github.actor != 'jonmeow'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete stale suggestions
+      - uses: Ardiannn08/resolve-outdated-comment@v1.1
```

Comment:
Why 1.1 when 1.3 is available?

https://github.com/Ardiannn08/resolve-outdated-comment

Also, please pin this dependency. You can use https://app.stepsecurity.io/secureworkflow for that.

Reply:
(the secureworkflow thing will also add other features, similar to what's done above)

Reply:
I was using 1.1 because that's what [the example uses](https://github.com/marketplace/actions/resolve-outdated-comments); changed to 1.3 and pinned.